src/ensembl/src/content/app/browser/browser-region-editor/BrowserRegionEditor.tsx

```diff
@@ -55,6 +52,7 @@ export const BrowserRegionEditor = (props: BrowserRegionEditorProps) => {
   const genomeKaryotype = props.genomeKaryotype as GenomeKaryotypeItem[];
   const [stick, locationStart, locationEnd] = props.chrLocation as ChrLocation;
 
+  const formRef = useRef(null);
```

Comment:
What does this ref do?

Reply:
It is not used anymore. I'll remove this.

Reply:
✅